pyuvdata/uvdata/mir.py

```diff
@@ -310,6 +311,13 @@ def read_mir(
         self.antenna_diameters = np.zeros(self.Nants_telescope) + 6
         self.blt_order = ("time", "baseline")
 
+        # set filename attribute
+        basename = filepath
+        while basename.endswith("/"):
+            basename = basename[:-1]
```

Comment:
I'm personally more of a fan of using rstip over this while statement.  It's also a little faster in these dummy test cases 
![image](https://user-images.githubusercontent.com/8646829/119360486-82f5e480-bc5f-11eb-8f64-37b396fe8e16.png)

```suggestion
       basename = basename.rstrip("/")
```

Reply:
Good suggestion, I've changed this in c194e2a.